site/content/integrate/outgoing-webhooks/_index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Outgoing Webhooks"
+date: "2017-08-19T12:01:23-04:00"
+section: "integrate"
+---
+
+# Outgoing Webhooks
+
+Outgoing webhooks let you receive an event as an HTTP POST when messages get posted into a Mattermost channel, with the option to respond with your own message.
+
+They can be configured to send events for one or both of the following conditions:
+
+* A message is posted in a specific channel
+* The first word of a message matches or starts with one of the configured trigger words
+
+### Basic Usage
+
+Follow [the admin guide](https://docs.mattermost.com/developer/webhooks-outgoing.html#create-an-outgoing-webhook) to create the webhook.
+
+After creating the webhook you'll be given a `token`. __Treat this as a secret.__ Anyone who has this token will be able to mimic requests to your application.
+
+Based on how you configured your webhook, the callback URL(s) you provided will receive HTTP POSTs similar to:
+
+```http
+POST /your-url HTTP/1.1
+Content-Length: 244
+Host: <your-host-name>
+Accept: application/json
+Content-Type: application/x-www-form-urlencoded
+
+channel_id=hawos4dqtby53pd64o4a4cmeoo&
+channel_name=town-square&
+team_domain=someteam&
+team_id=kwoknj9nwpypzgzy78wkw516qe&
+post_id=axdygg1957njfe5pu38saikdho&
+text=some+text+here&
+timestamp=1445532266&
+token=zmigewsanbbsdf59xnmduzypjc&
+trigger_word=some&
+user_id=rnina9994bde8mua79zqcg5hmo&
+user_name=somename
+```
+
+If you configured your webhook to use `application/json` content, the request would be:
+
+```http
+POST /your-url HTTP/1.1
+Content-Length: 378
+Host: <your-host-name>
+Accept: application/json
+Content-Type: application/json
+
+{
+  "channel_id": "hawos4dqtby53pd64o4a4cmeoo",
+  "channel_name": "town-square",
+  "team_domain": "someteam",
+  "team_id": "kwoknj9nwpypzgzy78wkw516qe",
+  "post_id": "axdygg1957njfe5pu38saikdho",
+  "text": "some text here",
+  "timestamp": "1445532266",
+  "token": "zmigewsanbbsdf59xnmduzypjc",
+  "trigger_word": "some",
+  "user_id": "rnina9994bde8mua79zqcg5hmo",
+  "user_name": "somename"
```

Comment:
I miss the file_ids field here.

Reply:
Oh yeah, forgot we sent that. I just copied this from the example on docs.mattermost.com. Nice catch, I'll add it